app/tests/test_regulations.py

```diff
@@ -909,3 +910,66 @@ def test_compute_regulations_per_week_not_enough_break(self):
         self.assertEqual(regulatory_alert[0].day, date(2022, 7, 18))
         extra_info = json.loads(regulatory_alert[0].extra)
         self.assertEqual(extra_info["rest_duration_s"], 111600)
+
+    def test_max_work_day_time_in_guyana_success(self):
+        company = self.company
+        admin = self.admin
+        how_many_days_ago = 2
+
+        GY_TZ_NAME = "America/Guyana"
```

Comment:
:orange_circle: La timezone qui correspond à la Guyane Française est America/Cayenne .

```suggestion
        GY_TZ_NAME = "America/Cayenne"
```